src/main/java/mate/controller/DeleteDriverController.java

```diff
@@ -0,0 +1,24 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.service.DriverService;
+
+@WebServlet(urlPatterns = "/drivers/delete")
+public class DeleteDriverController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate");
+    private final DriverService driverService
+            = (DriverService) injector.getInstance(DriverService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        Long id = Long.valueOf(req.getParameter("id"));
+        driverService.delete(id);
```

Comment:
```suggestion
        driverService.delete(id);
        resp.sendRedirect("/drivers");
```

lets show list of all drivers after we delete one

Reply:
done